ugvc/mrd/srsnv_plotting_utils.py

```diff
@@ -140,9 +183,10 @@ def plot_ROC_curve(
 
     recall = {}
     precision = {}
+
     for xvar, querystr, namestr in (
         ("X_SCORE", "X_SCORE>-1", "X_SCORE"),
```

Comment:
Please use the enumeration from featuremap_utils instead of the string "X_SCORE"